pkg/client/unversioned/remotecommand/v2.go

```diff
@@ -207,3 +186,9 @@ func (p *streamProtocolV2) stream(conn streamCreator) error {
 	// waits for errorStream to finish reading with an error or nil
 	return <-errorChan
 }
+
+type errorDecoderV2 struct{}
```

Comment:
nice to have: godoc explaining the variance between the different error decoders